lockfile/src/parsers/go_sum.rs

```diff
@@ -1,16 +1,27 @@
 use nom::branch::alt;
 use nom::bytes::complete::{tag, take_until};
 use nom::character::complete::{alphanumeric1, line_ending, space0, space1};
-use nom::combinator::{opt, recognize};
-use nom::multi::many1;
+use nom::combinator::recognize;
+use nom::multi::{many0, many1};
 use nom::sequence::{preceded, tuple};
 use phylum_types::types::package::PackageType;
 
-use super::IResult;
+use crate::parsers::{take_till_line_end, IResult};
 use crate::{Package, PackageVersion};
 
 pub fn parse(input: &str) -> IResult<&str, Vec<Package>> {
-    let (input, mut pkgs) = many1(package)(input)?;
+    let (input, lines) = many0(preceded(space0, take_till_line_end))(input)?;
+
+    let mut pkgs = Vec::new();
+    for line in lines {
+        if line.contains("/go.mod") {
+            continue;
+        }
+
+        if let Ok((_, pkg)) = package(line) {
+            pkgs.push(pkg);
+        }
+    }
 
     // Filter duplicate packages.
```

Comment:
Do we still need to filter out duplicates? I think this was only here to account for having a hash of the package and a hash of `/go.mod`

Reply:
Ah yes, that's correct, I'll remove.